nikola/conf.py.in

```diff
@@ -1046,8 +1046,17 @@ MARKDOWN_EXTENSIONS = ['markdown.extensions.fenced_code', 'markdown.extensions.c
 
 
 # Extra options to pass to the pandoc command.
-# by default, it's empty, is a list of strings, for example
+# It should have type Union[List[str], Dict[str, List[str]]]
```

Comment:
Writing out a type specification might be intimidating for users. (I’ll rewrite this comment slightly before merging.)

Reply:
deleted that line.